templates/tydiqa/secondary_task/templates.yaml

```diff
@@ -0,0 +1,135 @@
+    jinja: "{% set _blank=[\"passage\", \"text\", \"text snippet\"]|choice %}\n{%\
+      \ set _position = [\"above\", \"following\"] |choice %}\n{% set lang = id.split('-')[0]\
+      \ %}\n{% if lang == \"english\" %}\n{% if  _position == \"above\" %}\n{{context}}\n\
+      {{\"\\n\"}}\n{% endif %}\nI know that the answer to \"{{question}}\" appears\
+      \ somewhere in the {{_position}} {{_blank}}. Can you answer the question?\n\
+      {% if  _position == \"following\" %}\n{{\"\\n\"}}\n{{context}}\n{% endif %}\n\
+      |||\n{{answers.text[0]}} \n{% endif %}"
+    name: simple_question_reading_comp_1
+    reference: Ask question with Context
+    task_template: true
+  05c99237-0e03-4ec9-95f3-dfde8ae25605: !Template
+    id: 05c99237-0e03-4ec9-95f3-dfde8ae25605
+    jinja: "{% set _blank2 = [\"title\", \"context\", \"topic\" ] %}\n{% set _blank1\
+      \ = [\"guess\", \"generate\", \"determine\"] %}\n{% set _blank=[\"passage\"\
+      , \"text\", \"text snippet\", \"info\"]|choice %}\n{% set _position = [\"above\"\
+      , \"following\"] |choice %}\n{% set lang = id.split('-')[0] %}\n{% if lang ==\
+      \ \"english\" %}\n{% if  _position == \"above\" %}\n{{context}}\n{{\"\\n\"}}\n\
+      {% endif %}\nCan you {{_blank1|choice}} the {{_blank2|choice}} of the {{_position}}\
+      \ passage.\n{% if  _position == \"following\" %}\n{{\"\\n\"}}\n{{context}}\n\
+      {% endif %}\n|||\n{{title}} \n{% endif %}"
+    name: title_generation
+    reference: Generate title from a passage
+    task_template: false
+  1f4728ba-b25e-450e-975f-6dc8c0cb4bb1: !Template
+    id: 1f4728ba-b25e-450e-975f-6dc8c0cb4bb1
+    jinja: "{% set _blank1 = [\"generate\", \"ask\", \"come up with\"] %}\n{% set\
+      \ _blank2 = [\"passage\", \"text\", \"text snippet\"] %}\n{% set _position =\
+      \ [\"above\", \"following\"] |choice %}\n{% set lang = id.split('-')[0] %}\n\
+      {% if lang == \"english\" %}\n{% if  _position == \"above\" %}\n{{context}}\n\
+      {{\"\\n\"}}\n{% endif %}\nCan you {{_blank1|choice}} a question from the {{_position}}\
+      \ {{_blank2|choice}}?\n{% if  _position == \"following\" %}\n{{\"\\n\"}}\n{{context}}\n\
+      {% endif %}\n|||\n{{question}} \n{% endif %}"
+    name: end_to_end_question_generation
+    reference: generate end-to-end question from a passage.
+    task_template: false
+  1f5ae792-3d9c-4b78-9b7c-eb7c4ddb49af: !Template
+    id: 1f5ae792-3d9c-4b78-9b7c-eb7c4ddb49af
+    jinja: "{% set _blank1 = [\"generate\", \"ask\", \"come up with\"] %}\n{% set\
+      \ _blank2 = [\"passage\", \"text\", \"text snippet\"] %}\n{% set _position =\
+      \ [\"above\", \"following\"] |choice %}\n{% set lang = id.split('-')[0] %}\n\
+      {% if lang == \"english\" %}\n{% if  _position == \"above\" %}\n{{context}}\n\
+      {{\"\\n\"}}\n{% endif %}\nCan you {{_blank1|choice}} a question from the {{_position}}\
+      \ {{_blank2|choice}} on \"{{answers.text[0]}}\" topic?\n{% if  _position ==\
```

Comment:
change to `a question on [...] topic` to `a question whose answer is "{{answers.text[0]}}"`
same for multiple templates

Reply:
Done.